### DISCUSSION TITLE
Notification for Google Chat

### DISCUSSION THREAD

**Discussion by amanpal108** [codemagic-ci-cd/codemagic-docs#2409] [Ideas]
### Use case

```Text
Build Notification
```

### Idea

```Text
Integration of Google Chat for build notification
```

**Comment by mikhail-tokarev** [comment]
Hi,

thank you for your request and using Codemagic! At this moment we are not planning adding Google Chat integration, however you can always use custom scripts to implement such logic. Please reference Discord integration as an example here https://docs.codemagic.io/integrations/discord-integration/
